docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider.md

```diff
@@ -165,6 +165,7 @@ global:
         clientId: <Client ID from step 1>
         audience: <Audience of your application>
         existingSecret: <Client secret from step 3>
+        tokenScope: "<Client ID from step 1>/.default"
```

Comment:
What is the corresponding environment variable to capture this?

Reply:
❓ Client ID is correct here, does that help with the env variable question?

Reply:
Since this is in a tabbed section on the page, I wasn't sure if there is an environment variable to capture the `tokenScope` value.

Reply:
Ah I see, in that case there isn't one that would capture the complete value here as such, so IMO this is the correct approach (for Microsoft)